compat.yaml

```diff
@@ -1,11 +1,11 @@
 imports:
```

Comment:
I think we do not need to import the plugin yaml, because most users will use:
```
imports:
  - plugin:cloudify-openstack-plugin
```
This will lead to duplicate types.

Reply:
sure, will remove it then